Helper/helper.ps1

```diff
@@ -246,38 +246,51 @@ function GetStats {
 
 function __merge_csvs {
     param (
-        [Parameter(Mandatory=$true)]
+        [Parameter(Mandatory = $true)]
         [string]$csvs_path,
-        [Parameter(Mandatory=$true)]
+        [Parameter(Mandatory = $true)]
         [string]$xlsx_name
-        )
+    )
 
-        $csvs = Get-ChildItem -Path $csvs_path\* -Include *.csv
-        $excelFileName = $Destination_Path + "\" + $xlsx_name
-        Remove-Item $excelFileName -ErrorAction Ignore
-        Write-Host "Creating: $excelFileName" -ForegroundColor Green
+    $csvs = Get-ChildItem -Path $csvs_path\* -Include *.csv
+    $excelFileName = $Destination_Path + "\" + $xlsx_name
+    Remove-Item $excelFileName -ErrorAction Ignore
+    Write-Host "Creating: $excelFileName" -ForegroundColor Green
 
-        foreach ($csv in $csvs) {
-            $worksheetName = $csv.Name.Replace(".csv","")
-            Import-Csv -Path $csv.FullName| Export-Excel -Path $excelFileName -WorkSheetname $worksheetName
-        }
+    foreach ($csv in $csvs) {
+        $worksheetName = $csv.Name.Replace(".csv", "")
+        Import-Csv -Path $csv.FullName | Export-Excel -Path $excelFileName -WorkSheetname $worksheetName
+    }
 }
 
 function merge_csvs {
 
     if (Get-Module -ListAvailable -Name ImportExcel) {
-        Write-Host "Module exists"
+        Write-Host "ImportExcel module is already installed, trying to merge CSVs..."
```

Comment:
To differentiate between good status and bad status we use the -ForegroundColor in write-host

so in this case I'd recommend -ForegroundColor Green